apps/snmptable.c

```diff
@@ -734,6 +734,7 @@ get_table_entries(netsnmp_session * ss)
                                     break;
```

Comment:
Please use the imperative mood for the commit message. See also https://cbea.ms/git-commit/.